tasks.py

```diff
@@ -222,7 +222,8 @@ def create_user(context, user="admin", nautobot_ver=NAUTOBOT_VER, python_ver=PYT
     DEFAULT_ENV["PYTHON_VER"] = python_ver
 
     context.run(
-        f"docker-compose -f {COMPOSE_FILE} -p {BUILD_NAME} run nautobot nautobot-server createsuperuser --username {user}",
+        f"docker-compose -f {COMPOSE_FILE} -p {BUILD_NAME} run nautobot nautobot-server createsuperuser \
+         --username {user}",
```

Comment:
```suggestion
        f"docker-compose -f {COMPOSE_FILE} -p {BUILD_NAME} run nautobot "
        f"nautobot-server createsuperuser --username {user}",
```

Though in the long term we should refactor the `tasks.py` to match the newer pattern we have in other plugins (e.g. https://github.com/nautobot/nautobot-plugin-netbox-importer/blob/develop/tasks.py, https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/develop/tasks.py) where we use some helper functions to reduce the need to state `docker-compose -f {COMPOSE_FILE} -p {BUILD_NAME} run nautobot` all over the place.